src/txadmission.cpp

```diff
@@ -127,6 +135,19 @@ void FlushTxAdmission()
 void EnqueueTxForAdmission(CTxInputData &txd)
 {
     LOCK(csTxInQ);
+    // If I have lots of deferred tx, its probably because there's too much volume, so defer new ones right away
+    if (txDeferQ.size() > 1000)
+    {
```

Comment:
I wonder if the 1000 should be tied to some other number , like txns per sec or the aveCommitBatchSize, so that the value floats up and down? not sure if that makes sense here.

Reply:
I thought about it.  In theory we could use the estimated fastfilter collision probability with the number of incoming tx/sec... and we'd get some extremely low number.  What I'm trying to say is that the collisions are so low and the volume so high, you could pick pretty much any number and be fine.  So I don't see the sense in calculating it.